server/libs/modules/components/csv-file/src/main/java/com/bytechef/component/csv/file/util/CsvFileReadUtils.java

```diff
@@ -105,11 +107,17 @@ public static Iterator<CSVRecord> getIterator(
             BufferedReader bufferedReader, ReadConfiguration configuration) throws IOException {
 
         Iterator<CSVRecord> iterator;
+        List<String> regexSpecials = Arrays.asList(".", "+", "*", "?", "^", "$", "(", ")", "[", "]", "{", "}", "|",
```

Comment:
I would use different name for this variable. I perfectly understand the goal. I think better name for reserved characters would make code more readable.